internal/pkg/docker/orchestrator/orchestrator.go

```diff
@@ -15,11 +15,9 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/docker/dockerengine"
 )
 
-// Scheduler manages running a Task. Only a single Task
-// can be running at a time for a given Scheduler. A Scheduler
-// can only be Start()-ed once; multiple calls to Start() will
-// break things.
-type Scheduler struct {
+// Orchestrator manages running a Task. Only a single Task
+// can be running at a time for a given Orchestrator.
+type Orchestrator struct {
```

Comment:
I'm just curious here really, this makes sense why you're changing the name from Scheduler, because we're not actually running any traditional OS scheduling algorithms here. What made you pick calling it orchestrator though? Is there some CS precedent for the name choice?

Reply:
hahaha no but it sounds neat :) it's orchestrating containers!!!